tests/test_pipeline.py

```diff
@@ -666,9 +666,9 @@ def test_run_diad_pipeline_gpu(
     assert "Path to data: /entry/imaging/data" in verbose_log_contents
     assert "Preview: (100:3101, 8:15, 0:26)" in verbose_log_contents
     assert "Data shape is (3001, 7, 26) of type uint16" in verbose_log_contents
-    assert "Global min -0.0326580" in verbose_log_contents
-    assert "Global max 0.037757" in verbose_log_contents
-    assert "Global mean 0.000327" in verbose_log_contents
+    assert "Global min -0.00915" in verbose_log_contents
+    assert "Global max 0.01134" in verbose_log_contents
+    assert "Global mean 0.00013" in verbose_log_contents
```

Comment:
What has caused these to need to be changed?

Reply:
that is because we use 3d outliers/median instead of 2d now.